packages/@atjson/source-html/src/annotations/a.ts

```diff
@@ -0,0 +1,11 @@
+import { InlineAnnotation } from '@atjson/document';
```

Comment:
If druthers were to be handed out and I might have mine, the filename would be the exported class name `anchor.ts`...